stingray/lightcurve.py

```diff
@@ -1361,16 +1485,18 @@ def _apply_gtis(self):
         """
         check_gtis(self.gti)
```

Comment:
Have we not checked any GTIs before? Like whenever `self.mask` was created? Just making sure this isn't double. :)

Reply:
Just noticed another problem here.
I usually run `apply_gtis` after changing the gtis by hand (e.g. calculating the common gtis between two light curves). I also transformed `gti` into a property, and the setter of the property also sets `_mask` to None, so that the next time the mask is created it gets re-calculated with the new gtis.

Reply:
Ok, cool. If you use `_apply_gtis`, would it be worth making it an outward facing method, i.e. `apply_gits`, rather than a private one? Presumably other people might want to use it, too?

Reply:
Yes, I don't even know why I set it as a private method in the first place :D.
Done.